service_storagefacility/app/services/storage/NodeService.scala

```diff
@@ -45,7 +44,10 @@ trait NodeService {
       mid: MuseumId,
       nodeId: StorageNodeDatabaseId,
       envReq: EnvironmentRequirement
-  )(implicit currUsr: AuthenticatedUser): Future[Option[EnvironmentRequirement]] = {
+  )(
+      implicit currUsr: AuthenticatedUser,
+      ec: ExecutionContext
```

Comment:
Why do you send around the `ExecutionContext` here and not just import the default from play?